ev3dev/core.py

```diff
@@ -458,7 +459,7 @@ def position_sp(self, value):
     @property
     def speed(self):
         """
-        Returns the current motor speed in tacho counts per second. Not, this is
+        Returns the current motor speed in tacho counts per second. Note, this is
```

Comment:
This is autogenerated from [this line](https://github.com/ev3dev/ev3dev-lang/blob/946c4622aa97fb4d7d2efd73f41d404db58dc7e5/autogen/spec.json#L131) in spec.json. The change should go there.